ecmascript/transforms/proposal/src/decorators.rs

```diff
@@ -355,7 +355,7 @@ impl Decorators {
                     vars: &mut vars,
                     constructor_this_mark: None,
                     is_static: method.is_static,
-                    folding_constructor: true,
+                    folding_constructor: false,
```

Comment:
Folding decorator seems not in constructor function, they are in the second parameter of `_decorate`, which is factory function for decorator.